python/cudf/cudf/_lib/aggregation.pyx

```diff
@@ -78,8 +78,12 @@ class Aggregation:
         )
 
     @classmethod
-    def nunique(cls):
-        return cls(pylibcudf.aggregation.nunique(pylibcudf.types.NullPolicy.EXCLUDE))
+    def nunique(cls, dropna=True):
+        if dropna:
+            null_policy = pylibcudf.types.NullPolicy.EXCLUDE
+        else:
+            null_policy = pylibcudf.types.NullPolicy.INCLUDE
```

Comment:
Suggesting this change because I see its used somewhere else in this file.

```suggestion
        null_policy = pylibcudf.types.NullPolicy.EXCLUDE
        if dropna else pylibcudf.types.NullPolicy.INCLUDE
```

Reply:
Sure thing. Changed to this